iris-mpc-cpu/src/hawkers/galois_store.rs

```diff
@@ -253,9 +256,23 @@ impl VectorStore for LocalNetAby3NgStoreProtocol {
 }
 
 impl LocalNetAby3NgStoreProtocol {
+    pub fn get_trivial_share(&self, distance: u16) -> Share<u16> {
+        let player = self.get_owner_index();
+        let distance_elem = RingElement(distance);
+        let zero_elem = RingElement(0_u16);
+
+        match player {
+            0 => Share::new(distance_elem, zero_elem),
+            1 => Share::new(zero_elem, distance_elem),
+            2 => Share::new(zero_elem, zero_elem),
+            _ => panic!("Invalid player index"),
+        }
+    }
+
     async fn graph_from_plain(
         &mut self,
         graph_store: &GraphMem<PlaintextStore>,
+        recompute: bool,
```

Comment:
Rename to `recompute_distances`?  The variable isn't used in a context that the long variable name is cumbersome, and it makes it clear at a glance what is recomputed.